containers/app/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG OPEN_DEVIN_BUILD_VERSION=dev
+ARG OPEN_OPENHANDS_BUILD_VERSION=dev
```

Comment:
We probably need to global rename this to `OPENHANDS_BUILD_VERSION`

Reply:
Let's do this in subsequent PR

Reply:
Fixed :)